packages/quizzes-dashboard/pages/quizzes/[quizId]/[page].tsx

```diff
@@ -6,19 +6,29 @@ import AllAnswers from "../../../components/quizPages/answers/all"
 import RequiringAttention from "../../../components/quizPages/answers/requiring-attention"
 import TabNavigator from "../../../components/TabNavigator"
 
-export const QuizPage = () => {
+export const QuizPage = (props: any) => {
   const router = useRouter()
   const page = router.query.page?.toString() ?? ""
 
+  const { queryParams } = props
+
   return (
     <>
       <TabNavigator />
       {page === "overview" ? <OverView /> : ""}
       {page === "edit" ? <EditPage /> : ""}
-      {page === "all-answers" ? <AllAnswers /> : ""}
-      {page === "answers-requiring-attention" ? <RequiringAttention /> : ""}
+      {page === "all-answers" ? <AllAnswers queryParams={queryParams} /> : ""}
+      {page === "answers-requiring-attention" ? (
```

Comment:
This would do the same thing:
```javascript
 {page === "answers-requiring-attention" && <RequiringAttention queryParams={queryParams} /> }
```
I understand that the code above is doing the same thing -- maybe we could refactor those lines as well?